e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -555,22 +555,26 @@ Scenario('local authority tries to submit without giving consent', async (I, cas
   I.seeInCurrentUrl('/submitApplication');
 });
 
-Scenario('local authority submits after giving consent @create-case-with-mandatory-sections-only', async (I, caseViewPage, submitApplicationEventPage, paymentHistoryPage) => {
+let feeToPay = '2055'; //Need to remember this between tests.. default in case the test below fails
+
+Scenario('local authority submits after giving consent @create-case-with-mandatory-sections-only', async (I, caseViewPage, submitApplicationEventPage) => {
   await caseViewPage.startTask(config.applicationActions.submitCase);
 
-  const feeToPay = await submitApplicationEventPage.getFeeToPay();
+  feeToPay = await submitApplicationEventPage.getFeeToPay();
   submitApplicationEventPage.seeDraftApplicationFile();
   submitApplicationEventPage.giveConsent();
 
   await I.completeEvent('Submit', null, true);
   I.seeEventSubmissionConfirmation(config.applicationActions.submitCase);
   caseViewPage.selectTab(caseViewPage.tabs.documents);
   I.see('New_case_name.pdf');
+});
 
+Scenario('HMCTS admin check the payment', async (I, caseViewPage, paymentHistoryPage) => {
```

Comment:
the first 2 lines does not really belongs to this test, especially as done by user from previous test